fedimint-cli/src/ng.rs

```diff
@@ -82,6 +97,83 @@ pub async fn handle_ng_command<D: IDatabase>(
 
             Ok(serde_json::to_value(amt).unwrap())
         }
+        ClientNg::LnPay { bolt11 } => {
+            let invoice = bolt11.clone();
+            let operation_id = bolt11.payment_hash().into_inner();
+            let ln_client = client
+                .get_module_client::<LightningClientModule>(LEGACY_HARDCODED_INSTANCE_ID_LN)
+                .unwrap();
+            let mut dbtx = client.db().begin_transaction().await;
+            let outgoing_ln_contract = ln_client
+                .create_outgoing_ln_contract(
+                    &mut dbtx.with_module_prefix(LEGACY_HARDCODED_INSTANCE_ID_LN),
+                    client.context(),
+                    invoice,
+                    rng,
+                )
+                .await?;
```

Comment:
Ideally you'd return a `ClientOutput` from this function, much like in the mint module:

https://github.com/fedimint/fedimint/blob/d9cb6c2d44f0f9e659a27e9502735c6fa2e00376/modules/fedimint-mint-client/src/lib.rs#L173-L182

You'd return the strongly typed variant (see above example) and then convert it to the dynamic one using `into_dyn`

https://github.com/fedimint/fedimint/blob/d9cb6c2d44f0f9e659a27e9502735c6fa2e00376/fedimint-client/src/transaction/builder.rs#L39-L52

when you use it in `ng.rs`, just like done here with the `mint_input`:

https://github.com/fedimint/fedimint/blob/d9cb6c2d44f0f9e659a27e9502735c6fa2e00376/fedimint-cli/src/ng.rs#L61-L67